src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -154,7 +154,8 @@ def create_webapp(cmd, resource_group_name, name, plan, runtime=None, startup_fi
             site_config.linux_fx_version = _format_fx_version(encoded_config_file, multicontainer_config_type)
 
     elif plan_info.is_xenon:  # windows container webapp
-        site_config.windows_fx_version = _format_fx_version(deployment_container_image_name)
+        if deployment_container_image_name:
```

Comment:
Is it a valid condition that deployment_container_image_name is None?

Reply:
this is an optional parameter on the command.